package.json

```diff
@@ -6,12 +6,15 @@
     "type": "git",
     "url": "https://github.com/foxglove/rosbag.git"
   },
-  "main": "dist/cjs/node/index.js",
-  "module": "dist/esm/node/index.js",
-  "browser": "dist/esm/web/index.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "browser": "dist/esm/index.js",
   "files": [
     "dist",
-    "src"
```

Comment:
Our typical pattern is to include `src` in the module in case consumers want to be adventurous and import the .ts files directly. If you don't think that is a good idea let's discuss it in a PR in the template repo and propagate everywhere rather than changing this one repo.

Reply:
I removed it - I think its a bad idea. src is full of test snapshots and other stuff. If folks want to go exploring the sources they can read the online src code.

Reply:
want to update https://github.com/foxglove/template-typescript/blob/main/package.json and our other packages then?